lib/timer.js

```diff
@@ -71,11 +80,12 @@ class Timer extends Component {
         return;
       }
       this.setState({remainingTime: remaining});
-    }, 1);
+    }, 100);
```

Comment:
Seems unnecessary to run this every millisecond when its only counting by seconds anyway. This makes it count every 100 milliseconds instead